lib/geocoder/lookups/here_suggest.rb

```diff
@@ -25,9 +25,18 @@ def results(query)
       end
     end
 
+    def query_url_here_options(query)
```

Comment:
`query_url_here_options` is to be used for default url options which are common across all lookups, for eg `apiKey` and `lang`. Can we remove this override and move logic to `query_url_params` ?

Reply:
Based on geocoder gem's latest code https://github.com/alexreisner/geocoder/blob/master/lib/geocoder/lookups/here.rb#L46 they have it for both `geocode` and `revgeocode`, if this option is supported for all apis let's move it to `HereLookupDefaultMethods#query_url_here_options` else to  `HereSuggest#query_url_params`